build.sh

```diff
@@ -4,18 +4,22 @@ cd "$(dirname "$0")"
 
 # Available variables
 # -------------------
-declare BASE_ONLY;       # -- only build the base image (OS_FLAVOUR) without shipping PrestaShop
-declare REBUILD_BASE;    # -- force the rebuild of the base image
-declare DRY_RUN;         # -- if used, won't really build the image. Useful to check tags compliance
-declare OS_FLAVOUR;      # -- either "alpine" (default) or "debian"
-declare PHP_VERSION;     # -- PHP version, defaults to recommended version for PrestaShop
-declare PS_VERSION;      # -- PrestaShop version, defaults to latest
-declare PUSH;            # -- set it to "true" if you want to push the resulting image
-declare SERVER_FLAVOUR;  # -- not implemented, either "nginx" (default) or "apache"
-declare TARGET_IMAGE;    # -- docker image name, defaults to "prestashop/prestashop-flashlight"
-declare TARGET_PLATFORM; # -- a comma separated list of target platforms (defaults to "linux/amd64")
-declare PLATFORM;        # -- alias for $TARGET_PLATFORM
-declare ZIP_SOURCE;      # -- the zip to unpack in flashlight
+declare BASE_ONLY;             # -- only build the base image (OS_FLAVOUR) without shipping PrestaShop
+declare REBUILD_BASE;          # -- force the rebuild of the base image
+declare DRY_RUN;               # -- if used, won't really build the image. Useful to check tags compliance
+declare OS_FLAVOUR;            # -- either "alpine" (default) or "debian"
+declare PHP_VERSION;           # -- PHP version, defaults to recommended version for PrestaShop
+declare PS_VERSION;            # -- PrestaShop version, defaults to latest
+declare PUSH;                  # -- set it to "true" if you want to push the resulting image
+declare SERVER_FLAVOUR;        # -- not implemented, either "nginx" (default) or "apache"
+declare TARGET_IMAGE;          # -- docker image name, defaults to "prestashop/prestashop-flashlight"
+declare TARGET_PLATFORM;       # -- a comma separated list of target platforms (defaults to "linux/amd64")
+declare PLATFORM;              # -- alias for $TARGET_PLATFORM
+declare ZIP_SOURCE;            # -- the zip to unpack in flashlight
+declare PRE_INSTALLED_MODULES; # -- install modules during zip installation
+declare CUSTOM_LABELS;         # -- only when PRIVATE : list of key=value pairs separated by a comma, for overriding official flashlight labels
+
+declare -A IMAGE_LABELS;
```

Comment:
"s/IMAGE_LABELS/TARGET_IMAGE_LABELS"

Reply:
`IMAGE_LABELS` should be declared as a local variable later in the script.
Please merge #main in your branch, and introduce the `--target-image-labels` CLI option as well